packages/Table/src/Table.styles.js

```diff
@@ -3,21 +3,23 @@ import tokens from "@paprika/tokens";
 import { fontSize } from "@paprika/stylers/lib/helpers";
 import * as types from "./types";
 
-export const Table = styled.table`
-  border: 1px solid ${tokens.border.color};
-  border-collapse: collapse;
+export const Table = styled.table(() => {
+  return css`
+    border: 1px solid ${tokens.border.color};
+    border-collapse: collapse;
 
-  &:focus {
-    outline: 0;
-  }
+    &:focus {
+      outline: 0;
+    }
 
-  & .is-highlighted-focus {
-    outline: 2px solid blue;
-  }
-  & .is-highlighted-idle {
-    outline: 2px solid transparent;
-  }
-`;
+    & .is-highlighted-focus {
+      outline: 2px solid blue;
```

Comment:
A little out of scope, but maybe a quick token swap in would be a quick win?

Probably `tokens.highlight.active.noBorder.borderColor` is the best fit, or else just `tokens.color.blue`.

Reply:
`tokens.highlight.active.noBorder.borderColor` wao, really complex name for a token, wondering the cognitive load to remember this name :D will replace it.